src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -346,7 +355,13 @@ private async Task<IAnonymizer> CreateAnonymizerAsync(CancellationToken cancella
             List<Tuple<string, string>> queryParametersList = new List<Tuple<string, string>>(sharedQueryParametersList);
             if (progress.ContinuationToken != null)
             {
-                queryParametersList.Add(Tuple.Create(KnownQueryParameterNames.ContinuationToken, progress.ContinuationToken));
+                var continuationToken = progress.ContinuationToken;
+                if (!Base64FormatRegex.IsMatch(continuationToken))
+                {
+                    continuationToken = Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(continuationToken));
```

Comment:
After looking through this again I'm kind of surprised we don't have a `ToBase64` string extension :)

Reply:
We could add that when we remove the check in the ExportJobRecord in a week or two.